Cython/Utils.py

```diff
@@ -54,6 +54,21 @@ def wrapper(*args):
     return wrapper
 
 
+# or clear_instance_method_caches
+def clear_method_caches(instance):
```

Comment:
`clear_instance_method_caches` vs `clear_method_caches`

Reply:
I think `clear_method_caches` is clear enough (no pun intended). The word "instance" doesn't add anything, IMHO.

Reply:
comment removed [here](https://github.com/cython/cython/pull/4241/commits/4974fa843a599eff29b02585415923fa78b76313)